examples/lock-app/lock-common/lock-app.matter

```diff
@@ -593,7 +593,6 @@ server cluster DoorLock = 257 {
   attribute access(write: manage) boolean enablePrivacyModeButton = 43;
   attribute access(write: administer) int8u wrongCodeEntryLimit = 48;
   attribute access(write: administer) int8u userCodeTemporaryDisableTime = 49;
-  attribute access(write: administer) boolean requirePINforRemoteOperation = 51;
```

Comment:
I am not sure I follow.  This is removing one attrbiute, and adding a bit to the feature map.  Was everything else listed in #18407 fixed already?

Reply:
Yep, the reported incorrectly interpreted feature map and that lead to some confusion. I've verified that all the mandatory attributes for declared features are enabled. Also, your yesterdays commit of `door-lock-cluster.xml` cleanup helped tremendously :)

Reply:
Ah, reporter being confused about the feature map explains a lot.  :)  Thank you!